src/main/java/caculator/StringCalculator.java

```diff
@@ -0,0 +1,51 @@
+package caculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+
+    public static int splitAndSum(String text) {
+        if (isBlank(text)) {
```

Comment:
isBlank는 비교적 간단한 로직으로 보여요. 😃
굳이 메서드로 추출하지 않아도 충분히 가독성이 좋을 것 같네요. 😃

```suggestion
        if (text == null || text.isEmpty()) {
```

Reply:
[b00e634](https://github.com/next-step/java-racingcar-kakao/pull/31/commits/b00e6343552c97debe53b5321f7421e9944baf7c)
피드백주신대로 수정하였습니다.